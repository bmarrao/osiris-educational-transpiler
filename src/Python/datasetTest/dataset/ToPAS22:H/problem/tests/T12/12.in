5
12 9
M uXHTw vHWj RIIT dh GqXhp osjH UXr xmq Rb VaC nfr
vHWj GqXhp
UXr osjH
RIIT uXHTw
osjH xmq
dh GqXhp
RIIT Rb
M VaC
nfr osjH
uXHTw nfr
12 59
r c b TOCct PGWg rGdoq qsh o nqY Wi ykiq QqaW
nqY PGWg
r Wi
QqaW ykiq
qsh Wi
c o
QqaW c
c rGdoq
QqaW TOCct
PGWg o
o nqY
PGWg r
QqaW PGWg
c nqY
c qsh
r QqaW
qsh b
nqY r
QqaW qsh
qsh o
rGdoq o
TOCct b
TOCct c
TOCct r
PGWg Wi
QqaW rGdoq
TOCct o
b QqaW
nqY qsh
rGdoq qsh
Wi b
b ykiq
b rGdoq
TOCct PGWg
ykiq r
r qsh
QqaW o
r b
ykiq Wi
Wi TOCct
PGWg rGdoq
r rGdoq
nqY rGdoq
ykiq c
r o
Wi QqaW
rGdoq Wi
o b
o ykiq
rGdoq TOCct
qsh TOCct
nqY Wi
Wi o
TOCct ykiq
ykiq PGWg
PGWg qsh
ykiq nqY
PGWg b
rGdoq ykiq
b nqY
12 23
A IMiA l YmL BcGKa Hwskf O L r qAKE Vm FV
Hwskf IMiA
L YmL
IMiA BcGKa
L r
BcGKa FV
IMiA r
Vm BcGKa
Vm IMiA
qAKE l
FV O
r qAKE
A qAKE
IMiA YmL
qAKE BcGKa
Vm FV
Vm qAKE
L Vm
A BcGKa
L FV
Hwskf FV
FV YmL
IMiA FV
A r
12 28
e CvGh SuVoe CpbwE Kh lM je WD UpQ Ha PZjJg D
CvGh CpbwE
UpQ PZjJg
CvGh UpQ
WD Ha
Kh UpQ
SuVoe e
e D
CpbwE e
CvGh SuVoe
PZjJg Kh
SuVoe Ha
je SuVoe
CvGh lM
PZjJg CvGh
Ha PZjJg
PZjJg WD
CvGh je
UpQ je
UpQ SuVoe
Kh e
lM Kh
Kh D
SuVoe lM
CpbwE lM
PZjJg lM
e lM
D UpQ
CpbwE D
12 4
DpO btc G n Mq apT Rh jAHKN HxMP XNP Syi gsvY
DpO Syi
jAHKN DpO
btc XNP
apT XNP
