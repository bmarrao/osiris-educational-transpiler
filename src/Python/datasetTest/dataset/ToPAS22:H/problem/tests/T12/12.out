Possivel: 202199040
Impossivel
Possivel: 27648000
Possivel: 16035840
Possivel: 324172800
